src/query/src/sql.rs

```diff
@@ -227,6 +241,97 @@ fn describe_column_semantic_types(
     ))
 }
 
+fn show_create_column_name(table_name: Vec<String>) -> VectorRef {
+    Arc::new(StringVector::from(table_name))
+}
+
+fn show_create_table_column_name(table_info: TableInfoRef) -> VectorRef {
+    let mut create_sql_column: Vec<String> = Vec::new();
+
+    let columns_schemas = table_info.meta.schema.column_schemas();
+    let primary_key_indices = &table_info.meta.primary_key_indices;
+    let table_name = &table_info.name;
+
+    create_sql_column.push(format!("CREATE TABLE {} ( ", table_name));
+    for (index, cs) in columns_schemas.iter().enumerate() {
+        create_sql_column.push(cs.name.clone());
+        create_sql_column.push(cs.data_type.name().to_string());
+        let default_val = if cs.is_nullable() {
+            "null".to_string()
+        } else if cs.default_constraint().is_none() {
+            "".to_string()
+        } else {
+            format!(
+                "default {}",
+                cs.default_constraint()
+                    .map_or(String::from(""), |dc| dc.to_string())
+            )
+        };
+        create_sql_column.push(default_val);
+        if index != columns_schemas.len() - 1 {
+            create_sql_column.push(",".to_string());
+        }
+    }
+
+    let mut create_sql_index: Vec<String> = Vec::new();
+    for (index, cs) in columns_schemas.iter().enumerate() {
+        if primary_key_indices.contains(&index) {
+            create_sql_index.push(format!("primary key({})", cs.name));
+            create_sql_index.push(",".to_string());
+        } else if cs.is_time_index() {
+            create_sql_index.push(format!("time index({})", cs.name));
+            create_sql_index.push(",".to_string());
+        } else {
+        };
```

Comment:
Some style suggestions:

```suggestion
        if primary_key_indices.contains(&index) {
            create_sql_index.push(format!("primary key({}), ", cs.name));
        } else if cs.is_time_index() {
            create_sql_index.push(format!("time index({}), ", cs.name));
        }
```

Reply:
And also capitalize all the keywords, like `PRIMARY KEY`, `ENGINE`, etc.

Reply:
I use Vec container to store ', 'so as to delete the last', 'later.